main.cc

```diff
@@ -2,11 +2,16 @@
 #include <boost/asio.hpp>
 #include "server.h"
 
-int main() {
+int main(int argc, char** argv) {
+  if(argc != 2) {
+    std::cerr << "Usage: " << argv[0] << " config_path" << std::endl;
+    return 1;
+  }
+
   // Creates server object to accept incoming client connections
   try {
     boost::asio::io_service io_service;
-    Server server(io_service, 8118);
+    Server server(io_service, argv[1]);
```

Comment:
If we don't have two arguments, which port number does it run?

Also - we should add in the README about the commands to run :)